apps/web/app/players/[id]/page.tsx

```diff
@@ -151,7 +151,10 @@ export default async function PlayerPage(props: PageProps) {
     : Ruleset.Osu;
 
   // Get the list of tournaments that the player has participated in
-  const playerTournaments = await getTournaments(decodedId, searchParams);
+  const playerTournaments = await getTournaments(decodedId, {
+    ...searchParams,
+    ruleset: Number(currentRuleset).toString(),
```

Comment:
I'm pretty sure this is just a number under the hood since it's a numeric enum, no need to cast to `Number`